React/tic-tac/.eslintrc

```diff
@@ -1,3 +1,20 @@
-{   
-    "extends": "wolox"
+{
+  "parser": "babel-eslint",
```

Comment:
What do these lines do?

Reply:
I started to change from editor to webstorm and I think I did damage with the eslint